src/containers/App.js

```diff
@@ -76,7 +76,7 @@ export default class App extends React.Component {
   render () {
     const {
       children, community, dispatch, leftNavIsOpen, notifierMessages, isMobile,
-      showModal
+      showModal, route: { socket }
```

Comment:
Noted! I don't understand childContextTypes yet, but I will research it and make this work that way :)